chart/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "ots.fullname" . }}
+  labels:
+    {{- include "ots.labels" . | nindent 4 }}
+data:
+  customize.yml: |
+  {{- range $k, $v := .Values.customizations }}
+    {{ $k }}: {{ tpl $v $ }}
+  {{- end }}
```

Comment:
Hmm in theory this could work as
```yaml
data:
  customize.yml:
    {{- .Values.customizations | toYaml | tpl . | nindent 4 }}
```
You'd need to test whether `toYaml` butchers the templating… …if you really need the templating. If not you could just do `| toYaml | nindent 4`…

Reply:
thanks, for you input.
I tried with `tpl` to make value reference possible, but dropped it for now